docs/docs/loading-data-asynchronously.md

```diff
@@ -0,0 +1,114 @@
+---
+id: loading-data-asynchronously
+title: Loading Data Asynchronously
+permalink: docs/loading-data-asynchronously.html
+prev: typechecking-with-proptypes.html
+---
+
+
```

Comment:
We need to explain the key concept somewhere early. We need to hammer home the idea that in React, *any* update means a change in the state. It doesn't have any special capabilities for handling AJAX: if you want to change the UI as a result of data arriving, you need to change the state.